### DISCUSSION TITLE
Welcome to spi_avip Discussions!

### DISCUSSION THREAD

**Discussion by muneebullashariff** [muneeb-mbytes/spi_avip#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by muneebullashariff** [comment]
@All - Please comment to this post so that I know who all have access to this discussion forum.

**Reply by Shwetapatil05** [reply]
hai muneeb, I have access to this discussion

**Comment by Nadeem-Ahmed-S** [comment]
Hey Muneeb! I have access to this discussion.
